src/components/Navbar.tsx

```diff
@@ -0,0 +1,113 @@
+import ExpandLessIcon from "@mui/icons-material/ExpandLess";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import SearchIcon from "@mui/icons-material/Search";
+import {
+  AppBar,
+  Box,
+  Button,
+  Menu,
+  MenuItem,
+  Paper,
+  SvgIconProps,
+  TextField,
+} from "@mui/material";
+import { useState } from "react";
+import logo from "../assets/logo.svg";
+
+const categories = [
+  "🍽️ All recipes",
+  "🥪 Breakfasts",
+  "🍲 Soups",
+  "🍔 Lunch",
+  "🥐 Baking",
+  "🧁 Desserts",
+  "🍹 Drinks",
+  "🍿 Snacks",
+  "🥗 Salads",
+];
+
+export const Navbar = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
```

Comment:
I don't like the names here, they are too vague for me - handle click of what? handle close of what? I know that these are the only handlers in this component but I need to scroll down to "HTML" code to see where they are attached